app/views/govuk_component/breadcrumbs.raw.html.erb

```diff
@@ -1,11 +1,14 @@
 <%
   breadcrumbs ||= []
+  include_home = true if include_home.nil?
```

Comment:
One of the longer term goals of components is to make them more generalised, usable on other GDS projects, in multiple templating languages (like govuk_template) *

I think the default-true arguments would be hard to convert into some of the simpler templating languages (like `mustache`). Could we switch it to something more explicitly negative, like `no_home=true`? 

( * think isn't documented on the repo/contributing guide yet, so it's not clear, we're going to be improving that soon )